src/models/neural_networks/base.py

```diff
@@ -116,6 +118,9 @@ def train(
         # initialize the model
         if self.model is None:
             x_ref, _ = next(iter(train_dataloader))
+            assert (
```

Comment:
hmmmm... why am i asserting that there has to exist static data?

Reply:
good question? have you tried running things without static data? does it break?

Reply:
removed